src/files/ThemeXml.ts

```diff
@@ -0,0 +1,149 @@
+import { Archive } from '../classes/Archive.ts';
+import { XmlFile } from '../classes/XmlFile.ts';
+import { FileMime } from '../enums.ts';
+import { QNS } from '../utilities/namespaces.ts';
+import { evaluateXPathToMap } from '../utilities/xquery.ts';
+import { create } from '../utilities/dom.ts';
+
+/**
```

Comment:
Nit; normally the things that are being referenced are defined before/above their reference. IMHO (and I'm very used to this convention) this code becomes more readable if you swap the definition order for these types right around!